docs/workflow/requirements/windows-requirements.md

```diff
@@ -22,35 +22,43 @@ git config --system core.longpaths true
 - Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/). The Community edition is available free of charge.
 
 Visual Studio 2019 installation process:
-- It's recommended to use 'Workloads' installation approach. The following are the minimum requirements:
-  - .NET Desktop Development with all default components.
-  - Desktop Development with C++ with all default components.
-- To build for Arm32 or Arm64, make sure that you have the right architecture-specific compilers installed. In the "Individual components" window, in the "Compilers, build tools, and runtimes" section:
-  - For Arm32, check the box for "MSVC v142 - VS 2019 C++ ARM build tools (v14.23 or newer)".
-  - For Arm64, check the box for "MSVC v142 - VS 2019 C++ ARM64 build tools (v14.23 or newer)".
+- It's recommended to use **Workloads** installation approach. The following are the minimum requirements:
+  - **.NET Desktop Development** with all default components,
+  - **Desktop Development with C++** with all default components.
+- The build tools (CMake, Ninja and Python) can be downloaded and installed separately (see detailed instructions in the [section below](#build-tools)) or by selecting the following **Individual Components**:
+  - **C++ CMake tools for Windows** (includes Ninja),
+  - **Python 3 32-bit** for building x86 target architecture,
```

Comment:
Is it really required to have 32-bit Python to build for x86 target architecture? Can we recommend 64-bit Python for both instead?

Separately installed 64-bit Python works fine for building both x64 and x86 targets.

Reply:
Good point - unfortunately, I have not explicitly checked this combination. I have verified 64-bit Python is required for building x64 targets; I'll check if it can handle x86 also. Thanks

Reply:
Well, it seems I had some mix-ups with different combinations of the Python components and build target architectures. I am now going to use a dedicated virtual machine to have guaranteed clean environments and thoroughly verify all the combinations (it seems mono libraries x64 build with 64-Python VS component tries to use 32-bit python executable from folder that does not exist).

I'll report the results here.

I apologize for any inconvenience, I should have verified the VS components are working properly before opening the issue 😊 Thank you for your patience and understanding.

Reply:
Removed 32-bit Python component, the 64-bit one works fine for building both x86 and x64 targets.